### DISCUSSION TITLE
Is support with Ubuntu 24.04 (LTS) planned?

### DISCUSSION THREAD

**Discussion by McBaumwolle** [abraunegg/onedrive#2706] [General]
I am aware the OS update released yesterday, but I was wondering if compatibility with that version is planned.

**Comment by nathannli** [comment]
i also had the same question. I gave the compile method a go and it turned out to be quite simple if you wanted the onedrive tool asap

**Comment by abraunegg** [comment]
@McBaumwolle , @nathannli 

Due to where development is at, Ubuntu 24.04 includes v2.4.25 which is the current stable release. Details of this package can be found here: https://packages.ubuntu.com/noble/onedrive

It usually takes ~3-4 weeks for the OpenSuSE Build Service to have a new version of Ubuntu available. Once this is available, this *should* be included, thus v2.4.25 packages will be available via the reflected documentation process here. @norbusan could you please monitor for when xUbuntu 24.04 is available on the OpenSuSE Build Service and add this in when it is available?

Due to where I am at with the development of v2.5.0 application version, this (OPenSuSE Build service supporting Ubuntu 24.04) may tie in nicely with when I finally push v2.5.0 for general release.

For the moment if you want to use v2.4.25, just use the Universe package as this matches the current application release.

Now .. if you want to be brave, and wade into the waters to self-compile, please read https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md to get started on doing that .. however ... if you are going to self-compile, rather than use v2.4.25 - test out the v2.5.0 version. Please read https://github.com/abraunegg/onedrive/discussions/2662 for details. There are massive performance improvements and new features in this version which many will benefit from.

The current release candidate (v2.5.0-rc1) is fairly solid, with v2.5.0-rc2 literally around the corner (in the next day or so).

**Reply by McBaumwolle** [reply]
Wow thank you for the detailed reply! I think I can go without OneDrive a few weeks until v2.5.0 is ready. 

Thank yo for all your hard work!

**Comment by norbusan** [comment]
@abraunegg I will check. Just in case I forget (since I don't use ubuntu), if someone can ping me in 2-3 weeks that would be great.

**Comment by norbusan** [comment]
Ohhh, it is already there. I just added it to the build matrix. The builds should be ready in a few minutes.

**Comment by norbusan** [comment]
xUbuntu_24.04

**Comment by abraunegg** [comment]
Awesome - thanks

**Comment by abraunegg** [comment]
@McBaumwolle , @nathannli
Please refer to the updated documentation on how to install the client on Ubuntu 24.04:
* https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-2/docs/install.md
* https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-2/docs/ubuntu-package-install.md

Existing documentation for 'master' (v2.4.25) will not be updated at this point in time and will be automatically updated when v2.5.0 is released.

**Reply by McBaumwolle** [reply]
Thank you so much omg!
